pkg/terraform/stages/ovirt/stages.go

```diff
@@ -65,21 +71,28 @@ func extractOutputHostAddresses(s stages.SplitStage, directory string, ic *types
 			return
 		}
 		controlPlaneIPs = make([]string, len(vmIDs))
+		logrus.Debug("Retrieving control plane IPs from VMs: %v", vmIDs)
```

Comment:
This will only print out the address to the `vmIDs` slice and not the contents of the slice.

Reply:
I think `vmIDs` -> `&vmIDs` should resolve this but I'm testing in CI, not locally so it might still be wrong.

I wasn't necessarily planning on merging this so I probably should have just used `Spew`

Reply:
It seems that I am out of date on what the formatter does anyway.
https://go.dev/play/p/MgH1wx_VfZ4